core/resources/shaders/polyline.vs

```diff
@@ -43,15 +52,17 @@ void main() {
 
     vec4 position = a_position;
 
+    v_color = a_color;
     v_texcoord = a_texcoord;
+    v_normal = u_normalMatrix * vec3(0.,0.,1.);
 
     {
         float width = a_extrude.z;
         float dwdz = a_extrude.w;
-        float dz =  u_map_position.z - abs(u_tile_origin.z);
-        // interpolate between zoom levels
+        float dz = u_map_position.z - abs(u_tile_origin.z);
+        // Interpolate between zoom levels
         width += dwdz * dz;
-        // scale to screen-space
+        // Scale to screen-space
```

Comment:
I might be misunderstanding this, but width should be in model (tile) space here right (just like position is in tile/model space)?

Even in Line: 71 `position.xy += a_extrude.xy * width` is doing a operation in model space.

And if width is in model space, does that mean injection for width is in model space?